crates/holochain/src/core/ribosome/host_fn/call.rs

```diff
@@ -122,11 +122,12 @@ pub mod wasm_test {
             ZomeCallResponse::Ok(guest_output) => {
                 let agent_info: AgentInfo = guest_output.decode().unwrap();
                 assert_eq!(
-                    agent_info,
-                    AgentInfo {
-                        agent_initial_pubkey: bob_agent_id.clone(),
-                        agent_latest_pubkey: bob_agent_id.clone(),
-                    },
+                    &agent_info.agent_initial_pubkey,
+                    bob_agent_id
+                );
+                assert_eq!(
+                    &agent_info.agent_latest_pubkey,
+                    bob_agent_id
```

Comment:
just to have a test looking at the chain head here, is the sequence number consistent, could you check for that?